tests/conditional_compilation/test_infer.py

```diff
@@ -4,14 +4,17 @@
 
 """ Test inference with conditional compiled binaries.
 """
-
 from proc_utils import cmd_exec  # pylint: disable=import-error
+from install_pkg import get_openvino_environment  # pylint: disable=import-error
 
 
-def test_infer(model, benchmark_app):
+def test_infer(test_id, model, artifacts):
     """ Test inference with conditional compiled binaries
     """
+    install_prefix = artifacts / test_id / "install_pkg"
+    benchmark_app = install_prefix / "bin" / "benchmark_app"
```

Comment:
`benchmark_app` may be not only binary, but .exe. It may be resolved like `f"benchmark_app{".exe" if os == Win else ""}"`